styles/skin.less

```diff
@@ -6,7 +6,7 @@
 	.buildSkinRules(@skin; @importRules) {
 		&:global(.@{skin}) {
 			// Establish a base set of colors and variables that other skins can optionally override.
-			@import "./colors-gallium.less";
+			@import "./colors-gallium-day.less";
 			@import "./variables-gallium.less";
```

Comment:
Should this be gallium-day as well?

Reply:
`variables` is independent of day/night; there's just the one.

Reply:
Understood. Might have a slight preference for making `gallium-day` be `gallium` if it's the default for consistent naming but not critical to resolve at the moment.

Reply:
Personally, I'd lean toward renaming `gallium` to `gallium-night` so it's clearer.

Reply:
Whichever is fine